courses/credentials.py

```diff
@@ -0,0 +1,92 @@
+"""Digital courseware credentials"""
+from typing import Dict, Union
+from urllib.parse import urljoin
+
+from django.conf import settings
+from mitol.common.utils import now_in_utc
+from mitol.digitalcredentials.models import LearnerDID
+
+from courses.models import (
+    CourseRun,
+    CourseRunCertificate,
+    ProgramCertificate,
+    ProgramRun,
+)
+
+
+def build_program_run_completion(program_run: ProgramRun) -> Dict:
+    """Build a completion object for a ProgramRun"""
+    return {
+        "type": "schema:EducationalOccupationalProgram",
+        "identifier": program_run.program.text_id,
+        "name": program_run.program.title,
+        "description": program_run.program.page.description,
+        "numberOfCredits": {"value": program_run.program.page.certificate_page.CEUs},
```

Comment:
Since we have some data coming from the CMS, can we raise an error if these happen to be empty?